src/components/Footer.css

```diff
@@ -0,0 +1,34 @@
+footer {
```

Comment:
Since we are component driven try to put the everything related to Footer in a Footer folder under /components